tests/cross_functional/system_test/test_mcg_replication_with_disruptions.py

```diff
@@ -404,3 +419,333 @@ def test_log_based_replication_with_disruptions(
             object_amount=5,
         )
         logger.info("No issues seen with the MCG bg feature validation")
+
+
+class TestMCGReplicationWithVersioningSystemTest:
+
```

Comment:
add docs strings

Reply:
doc strings are present under the test function